sender-stackdriver/src/main/java/zipkin2/reporter/stackdriver/StackdriverSender.java

```diff
@@ -55,6 +55,7 @@ public static Builder newBuilder(Channel channel) { // visible for testing
     CallOptions callOptions = DEFAULT;
     boolean shutdownChannelOnClose;
     long serverResponseTimeoutMs = DEFAULT_SERVER_TIMEOUT_MS;
+    String expectedHealthcheckError = "INVALID_ARGUMENT";
```

Comment:
Is it possible to cast the exception to StatusException or such to extract the actual gRPC status code?

Reply:
Yes; much better than comparing text.